deploy/docker-compose-1AER_8AEO.yml

```diff
@@ -0,0 +1,125 @@
+    command: bash -c "source /work/build/Release/generators/conanrun.sh && export LD_LIBRARY_PATH=$${LD_LIBRARY_PATH}:/work/build/Release/lib && /work/build/Release/bin/JM_PROC_transient -log-config config/log-pv-kafka.properties -configFile ${CONFIG_FILE:-pv-config.json} -util Kafka -kafka-broker-list kafka:9093 -kafka-group-id PVKafkaGroup -util MetaData"
+
+  istore_proc:
+    image: levistarrett/masl-dev:${MASL_VERSION:-latest}
+    depends_on:
+      kafka:
+        condition: service_healthy
+    volumes:
+      - type: volume
+        source: ConanCache
+        target: /conan-cache
+      - "../models/PV_PROC/build:/work/build"
+      - "../models/PV_PROC/schedule:/root/schedule"
+      - "./config:/root/config"
+      - "./InvariantStore:/root/InvariantStore"
+    ulimits:
+      sigpending: -1
+    logging:
+      driver: local
+    working_dir: /root
+    command: bash -c "source /work/build/Release/generators/conanrun.sh && export LD_LIBRARY_PATH=$${LD_LIBRARY_PATH}:/work/build/Release/lib && /work/build/Release/bin/ISTORE_PROC_sqlite -db ./InvariantStore/InvariantStore.db -log-config config/log-pv-kafka.properties -util Kafka -kafka-broker-list kafka:9093 -kafka-offset-reset latest -util MetaData"
+
+  aeo_svdc_proc:
+    image: levistarrett/masl-dev:${MASL_VERSION:-latest}
+    deploy:
+      replicas: 8
+    depends_on:
+      kafka:
+        condition: service_healthy
+    volumes:
+      - type: volume
+        source: ConanCache
+        target: /conan-cache
+      - "../models/PV_PROC/build:/work/build"
+      - "../models/PV_PROC/schedule:/root/schedule"
+      - "./config:/root/config"
+    ulimits:
+      sigpending: -1
+    logging:
+      driver: local
+    working_dir: /root
+    command: bash -c "source /work/build/Release/generators/conanrun.sh && export LD_LIBRARY_PATH=$${LD_LIBRARY_PATH}:/work/build/Release/lib && /work/build/Release/bin/AEO_SVDC_PROC_transient -log-config config/log-pv-kafka.properties -configFile ${CONFIG_FILE:-pv-config.json} -util Kafka -kafka-broker-list kafka:9093 -kafka-offset-reset latest -util MetaData"
+
+  http_server:
+    image: "ghcr.io/xtuml/charon:v0.1.0"
+    environment:
+      - GIN_MODE=release
+    volumes:
+      - "./config/job_definitions:/data/aeo_svdc_config/job_definitions"
+      - "./logs/protocol_verifier:/data/logs/verifier"
+      - "./logs/protocol_verifier:/data/logs/reception"
+    ports:
+      - 9000:9000
+    command: "-path=/data"
+
+  zookeeper:
+    image: levistarrett/zookeeper:3.4.6
+    ports:
+      - "2181:2181"
+
+  kafka:
+    image: levistarrett/kafka:${PV_HOST_CPUTYPE:-latest}
+    ports:
+      - "9092:9092"
+    expose:
+      - "9093"
+    healthcheck:
+      test: /opt/kafka/bin/kafka-topics.sh --bootstrap-server kafka:9093 --topic Protocol_Verifier_Reception,AEOrdering_service0,IStore_service0 --describe
+      interval: 10s
+    environment:
+      KAFKA_ADVERTISED_LISTENERS: INSIDE://kafka:9093,OUTSIDE://172.31.13.129:9092
```

Comment:
Do we want to have this IP here? Maybe we could have a placeholder like local host (think it would be good to have documented somewhere that you change this value depending on where you want kafka to advertise on)